app/javascript/react/screens/App/Overview/screens/PlanWizard/__tests__/index.test.js

```diff
@@ -21,7 +21,11 @@ describe('Plan Wizard integration test', () => {
     createStore(
       combineReducers({ ...reducers }),
       {
-        overview: initialState
+        overview: initialState,
+        form: {
```

Comment:
if you check latest, @michaelkro found a better fix for this last night...

Reply:
Ok, I'll look!